### PR TITLE
UCP: correct the messages about the available UCX_TLS.

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Comment by alinask** [comment]
@yosefe please review

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1392/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/780/ for details (Mellanox internal link).

File: src/ucp/core/ucp_context.c [hunk 1]
```diff
@@ -40,9 +40,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "In addition it's possible to use a combination of the following aliases:\n"
    " - all    : use all the available transports.\n"
    " - sm/shm : all shared memory transports.\n"
+   " - mm     : shared memory transports - only memory mappers.\n"
    " - ugni   : ugni_rdma and ugni_udt.\n"
-   " - rc     : rc and cm.\n"
-   " - rc-x   : rc with accelerated verbs and cm.\n"
+   " - rc     : rc and ud.\n"
+   " - rc-x   : rc with accelerated verbs and ud.\n"
```
Comment (miked-mellanox) [inline src/ucp/core/ucp_context.c]: why "and ud" for rc-x?

Comment (alinask) [inline src/ucp/core/ucp_context.c]: becuase ud is used for connection establishment.
if you run with: "-x UCX_TLS=*rc_x* -x UCX_NET_DEVICES=mlx5_0:1", you still get:
selected for runtime: rc_mlx5/mlx5_0:1 [2->2] pd 0
selected for auxiliary: *ud*/mlx5_0:1 [0->0] pd 0

**Comment by yosefe** [comment]
:+1:

File: src/ucp/core/ucp_context.c (hunk 1 above)
Comment (miked-mellanox) [inline src/ucp/core/ucp_context.c]: maybe we should emphasize that ud in this context selected implicitly and used for OOB.

**Pull request merged**